wire-ios-request-strategy/Sources/Request Strategies/Connection/ConnectionRequestStrategy.swift

```diff
@@ -35,10 +35,14 @@ public class ConnectionRequestStrategy: AbstractRequestStrategy, ZMRequestGenera
     let connectToUserActionHandler: ConnectToUserActionHandler
     let updateConnectionActionHandler: UpdateConnectionActionHandler
     let actionSync: EntityActionSync
+    let oneOnOneResolver: OneOnOneResolverInterface?
```

Comment:
suggestion: make this non optional and provide a default value if nil is passed in the init.

Reply:
Done!